tendrl/gluster_integration/sds_sync/__init__.py

```diff
@@ -79,7 +79,9 @@ def _run(self):
                         lock.acquire(lock_ttl=60)
                 except etcd.EtcdLockExpired:
                     continue
-                gevent.sleep(10)
+                gevent.sleep(
+                    int(NS.config.data.get("sync_interval", 10))
```

Comment:
Actually at the moment, there are no ttl set for any entities in gluster-integration. Later if @GowthamShanmugam 's out of box delete PR needs ttl, he can use the logic to set the value as `sync_interval + 15`.

@GowthamShanmugam ^^^